### ISSUE TITLE
Betting: Leave room confirmation enhancements

### ISSUE THREAD

**Issue by madhav-halbe** [sidebetz/feedback#1]
When a user clicks the leave room icon, a window appears which asks the user for leave room confirmation. We should have the user select from 2 options: 

1. Temporarily leave the tournament
2. Permanently leave the tournament.

The flow for above option # 1 is what is done currently.

If the user selects option # 2, then transfer all the tournament chipz/balance/wallet to the main wallet. 

If the user has placed bets on the current hand which is not over, then the tournament balance after the end of the hand is what will be transferred to the main wallet.

**Comment by duck-tape** [comment]
This ticket has backend needs and then front end hence first assigning it to @JavierFerrerGomez and then will move to @achampagnedev

**Comment by milmanDev** [comment]
Related to #417

**Comment by scarletbeast** [comment]
The code is completed, and needs to be approved and reviewed.

This is related to [https://github.com/sidebetz/sidebetz/pull/473](https://github.com/sidebetz/sidebetz/pull/473)

Also can be tested here: [https://sb-next-bryan-leave-tournament-permanently-endpoi-26pj3lhfwq-as.a.run.app/](https://sb-next-bryan-leave-tournament-permanently-endpoi-26pj3lhfwq-as.a.run.app/)

**Comment by scarletbeast** [comment]
Toaster when pending bets are present before leaving tournament permanently..  After the toaster goes away, it redirects to main lobby/homepage  (and also opens feedback_url link for beta user feedback collected by google forms).

Pending bets are automatically settled via the **/handended** api in main lobby when the hand is over. 
**Note:** Currently the chipz balance in main lobby doesn't refresh automatically.   This was considered out of scope by me and documented here:  This is related to [#486](https://github.com/sidebetz/sidebetz/issues/486)

![image](https://github.com/user-attachments/assets/49572dae-e19e-4b46-b8fb-2b76e3875780)

Per Alex: the toaster doesnt vertically align centered with overlay buttons text, but he is going to probably remove the toaster anyway soon.

**Comment by scarletbeast** [comment]
Reassigning to @JavierFerrerGomez for code review, and then assigning to @madhav-halbe  once code is merged for UAT.

**Comment by JavierFerrerGomez** [comment]
@scarletbeast review is done, reassigned to you

**Comment by JavierFerrerGomez** [comment]
@achampagnedev can you help me test this? I need a second pair of eyes. If you want to see at the TS code check this commit: 
https://github.com/sidebetz/sidebetz/pull/571/commits/1231f586da1ab4ce8ce97c83c7146bdaac27c7d4

**Comment by milmanDev** [comment]
Validated on Dev. Working as expected. 

Moving to Done.
